lte/gateway/c/core/oai/tasks/mme_app/mme_app_procedures.cpp

```diff
@@ -38,7 +38,7 @@ extern "C" {
 #endif
 
```

Comment:
can you move https://github.com/magma/magma/blob/f8cf7bcb4204d4f0dee22cebe7daed6bd14f61f3/lte/gateway/c/core/oai/tasks/mme_app/mme_app_procedures.cpp#L47 to the top